check.sh

```diff
@@ -10,7 +10,7 @@ export JAVAC=$CF/checker/bin/javac
 export PICO=$(cd $(dirname "$0") && pwd)
 
 # Dependencies
-export CLASSPATH=$PICO/build/classes/java/main:$CFI/dist/checker-framework-inference.jar
+export CLASSPATH=$PICO/build/libs/immutability.jar:$CFI/dist/checker-framework-inference.jar
```

Comment:
I think we want `$PICO/build/classes/java/main` on the `CLASSPATH` so that we don't have to always rebuild the `immutability.jar` file. When editing in Eclipse the folder will contain newer `.class` files and it is easy to forget rebuilding the `.jar` file.
So simply keep `$PICO/build/classes/java/main` as the first entry. Is there maybe another directory that you can add for the `messages.properties` file? So you would have three entries instead of one: one for `.class` files, one for the `messages.properties` file, and finally the `immutability.jar` file.